script/QA/QA_18.txt

```diff
@@ -11,12 +11,14 @@ Do you have a sufficient amount of hemoglobin?
 [sha:792df296959e52d34da2d7e08b63dacef72697e1]{
 -- Page 113, Offset 43523.
 -- “そういう結末もありか”と
-The corner where we cleanse your palate after you proclaim,
+You're due a break and a palate cleanser%{n}
+after
 }
 [sha:f84fd291bb08e2d72b42283fffe20a22ad45d33e]{
 -- Page 113, Offset 43524.
 -- 一息ついた後の口直しコーナー、
-'So this is how it ends' with your last breath.
+ your discovery that endings such as%{n}
+this also exist.
```

Comment:
I think them introducing the Q&A with a different name for the corner is sort of a running gag? Should be there.